kolibri/core/content/utils/paths.py

```diff
@@ -119,10 +119,20 @@ def get_content_server_url(path, baseurl=None):
     return urljoin(baseurl, path)
 
 
-def get_channel_lookup_url(identifier=None, baseurl=None):
-    content_server_path = "/api/public/v1/channels"
+def get_info_url(baseurl=None):
+    return get_content_server_url('/api/public/info', baseurl=baseurl)
+
+
+def get_channel_lookup_url(version='1', identifier=None, baseurl=None, keyword=None, language=None):
+    content_server_path = '/api/public/v{}/channels'.format(version)
     if identifier:
-        content_server_path += "/lookup/{}".format(identifier)
+        content_server_path += '/lookup/{}'.format(identifier)
+    content_server_path += '?'
+    if keyword:
+        content_server_path += 'keyword={}&'.format(keyword)
+    if language:
+        content_server_path += 'language={}'.format(language)
```

Comment:
Would be much cleaner to use `urlencode` here. Note that to avoid special Py2/3 logic, you can just do:
`from django.utils.http import urlencode`

```
>>> urlencode({"qqq": 7, "qaaz": "hi"})
'qqq=7&qaaz=hi'
```

Reply:
cool!